cmd/neofs-cli/modules/control/object_list.go

```diff
@@ -0,0 +1,55 @@
+package control
+
+import (
+	rawclient "github.com/nspcc-dev/neofs-api-go/v2/rpc/client"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/common"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/commonflags"
+	"github.com/nspcc-dev/neofs-node/cmd/neofs-cli/internal/key"
+	"github.com/nspcc-dev/neofs-node/pkg/services/control"
+	"github.com/spf13/cobra"
+)
+
+var listObjectsCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List of all objects in the storage node",
+	Long:  "List of all objects in the storage node",
+	Args:  cobra.NoArgs,
+	Run:   listObjects,
```

Comment:
we are trying to move to `RunE` (i hope, https://github.com/nspcc-dev/neofs-node/issues/2890), can you, please, start to use it here too?

Reply:
I was based on this #623, but I don't think I understood.

Reply:
ok, since it is not discussed, let it be `Run` then